src/cli.rs

```diff
@@ -244,12 +247,16 @@ pub struct Cli {
     pub method: Option<Method>,
 
     /// The request URL.
-    #[structopt(skip)]
-    pub url: String,
+    #[structopt(skip = ("http://placeholder".parse::<Url>().unwrap()))]
```

Comment:
Is this needed because Url doesn't implement the default trait?

Reply:
Yes.